systems/registry/api-gateway/pkg/rest/router.go

```diff
@@ -238,6 +248,26 @@ func (r *Router) getSiteNodesHandler(c *gin.Context, req *GetSiteNodesRequest) (
 func (r *Router) getNodes(c *gin.Context) (*nodepb.GetNodesResponse, error) {
 	return r.clients.Node.GetNodes()
 }
+func (r *Router) list(c *gin.Context, req *ListNodesRequest) (*nodepb.ListResponse, error) {
+	listReq := &nodepb.ListRequest{
+		Type:         req.Type,
+		SiteId:       req.SiteId,
+		NodeId:       req.NodeId,
+		NetworkId:    req.NetworkId,
+		State:        Undefined,
```

Comment:
Why are we using **undefined** constant when we already have `ukama.NodeStateUnknown `

Reply:
`ukama.NodeStateUnknown` have its own meaning. i.e: Node is ready to configure